### PR TITLE
[MIG] shopinvader_locomotive: Migration to 14.0

### PR DESCRIPTION
**Pull request by simahawk**
Replaces #859

And fwd porting  from v13: 3d1e123 and 3588e30 and a small wording fix.

### CODE REVIEW

**Comment by simahawk** [comment]
@PierrickBrun @rousseldenis as the other PR had no review, could you pass by here?

**Review by PierrickBrun — APPROVED** [review]

File: shopinvader_locomotive/component/shopinvader_site_export_mapper.py [hunk 1]
```diff
@@ -47,11 +48,13 @@ def country(self, record):
         }
 
     @mapping
-    @changed_by("filter_ids")
+    @changed_by("visible_filter_ids")
     def filters(self, record):
         return {
             "all_filters": self._m2m_to_external(
-                record, "filter_ids", ["name", "display_name:code", "help"]
```
Comment (rousseldenis) [inline shopinvader_locomotive/component/shopinvader_site_export_mapper.py:54]: :smiley: I skipped some devs as I wasnt't aware that locomotive filters can be managed through Odoo! Cool

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by simahawk** [comment]
merci!

/ocabot merge patch

**Pull request merged**